Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -363,6 +364,18 @@ static void OnLoad()
                     },
                     analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.ShaderCompilerMessages
                 });
+
+                ViewDescriptor.Register(new ViewDescriptor
+                {
+                    category = IssueCategory.Texture,
+                    name = "Texture",
+                    menuLabel = "Texture/Diagnostics",
+                    menuOrder = 6,
+                    showInfoPanel = true,
+                    analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.Texture
+                });
```

Comment:
please make sure that double-clicking on the table item, the texture is shown in the inspector. I think you just need to add
`onOpenIssue = EditorUtil.FocusOnAssetInProjectWindow,`
here in the ViewDescriptor registration.

Reply:
Added, and also add a .WithLocation(location);  and location var to the TestModule.cs script in the ProjectIssue.Create portion as I was not actually reporting/sending along the location with each issue. 
Works!